homeassistant/components/ps4/media_player.py

```diff
@@ -96,7 +96,6 @@ def __init__(
         self._retry = 0
         self._disconnected = False
 
-    @callback
     def status_callback(self) -> None:
```

Comment:
Can't comment on the actual line below, line 103: `self.async_write_ha_state()`.
I think we should use `self.schedule_update_ha_state` there, since this is run in the executor now and not the event loop, right?